lms/static/js/edxnotes/views/tab_view.js

```diff
@@ -116,9 +116,7 @@ define([
         showErrorMessage: function (message) {
             this.$('.wrapper-msg')
                 .removeClass('is-hidden')
-                .find('.msg-content .copy').text(message)
-                .closest('.msg')
-                .focus();
```

Comment:
@cptvitamin, since this is just an error message, I'm very fine with that approach (using ``aria-live`` values). Does the ``polite`` value need to be toggled when a message is shown? If not, I can make the quick HTML change. If so, @polesye, is this something you can help wire up?